skimage/filters/_gaussian.py

```diff
@@ -155,3 +155,132 @@ def _guess_spatial_dimensions(image):
         return 3
     else:
         raise ValueError("Expected 2D, 3D, or 4D array, got %iD." % image.ndim)
+
+
+def difference_of_gaussians(image, lesser_sigma, greater_sigma=None, *,
```

Comment:
I still think `high` and `low` are better choices here. One debate that we are [having now](https://github.com/scikit-image/meeting-notes/blob/master/2020/2020-02-17.markdown) is how best to make all APIs across scikit-image consistent, including parameters names. I think high and low or max and min are much more common in the API currently. Anyway, `sigma` is a common notation in image domain, rather than frequency domain, where people typically use theta. So, I think the risk of confusing high and low with the frequency band is very low.